keycloak_client.go

```diff
@@ -413,7 +413,7 @@ func whitelistErrors(statusCode int, message string) error {
 			Message: "keycloak." + message,
 		}
 	// update account in back-office or self-service
-	case (message == "User exists with same username or email") || (message == "usernameExistsMessage") || (message == "emailExistsMessage"):
+	case (message == "User exists with same username or email") || (message == "usernameExistsMessage") || (message == "emailExistsMessage") || (message == "Username exists with same username") || (message == "Username exists with same email"):
```

Comment:
This line starts to be a little bit too long...
You should use a static map with white-listed messages as keys

Reply:
But... Will I receive an "error code" as well, something like keycloak-bridge.username-exists ?
Or will I receive a full sentence in english ?

Reply:
you will not receive a full sentence, nothing changed in how to format our message errors

Reply:
@fperot74 , ok now?

Reply:
great :-)
Can you give me the list of all "new error codes" I could receive?
thanks!

Reply:
it should be keycloak.existing.UsernameOrEmail